options.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+	const saveBtn = document.getElementById('saveBtn');
+
+	if (saveBtn) {
```

Comment:
Don't have a save button. It should just save live. See: https://github.com/sindresorhus/notifier-for-github-chrome/blob/994bae39590596754fbc9c482663342e51999791/extension/options.js#L35